### PR TITLE
[RHPAM-3325] reconcile error due to mem conversion and startupProbe

### PR DESCRIPTION
**Pull request by tchughesiv**
Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: pkg/controller/kieapp/constants/constants.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ const (
 	ConsoleAuthoringCPURequests = "1500m"
 	ConsoleAuthoringMemRequests = "3Gi"
 	ConsoleProdCPURequests      = "500m"
-	ConsoleProdMemRequests      = "1.5Gi"
+	ConsoleProdMemRequests      = "1500m"
```
Comment (spolti) [inline pkg/controller/kieapp/constants/constants.go:176]: shouldn't it be 1500Mi or keep as 1.5Gi?

Comment (tchughesiv) [reply pkg/controller/kieapp/constants/constants.go:176]: We can be, I was just matching the other memory settings. Your call though

Comment (tchughesiv) [reply pkg/controller/kieapp/constants/constants.go]: done

**Pull request merged**